src/main/java/task/type/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package task.type;
+
+public class Deadline extends Task {
+    protected String by;
+    public static final String type = "D";
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+    public Deadline(String description, String by, boolean isDone) {
+        super(description, isDone);
+        this.by = by;
+    }
+
+    public String getBy(){
+        return by;
+    }
+    @Override
+    public String toString() {
+        return "[" + type + "]" + super.toString() + " (by: " + by + ")";
+    }
+
```

Comment:
You can try to be more consistent with how much spacing you have between functions. Having spaces between your functions makes it clearer and easier to read as well